### ISSUE TITLE
ec2-user does not have permission to newly created worlds

### ISSUE THREAD

**Issue by cat-box** [cat-box/aws-foundry-ssl#5]
Current workarounds: 

1. Create a copy of the world folder and delete the original world folder
2. SSH using putty and elevate to root to change permissions or sftp files and folders

We will rehaul the setup method and permissions to fix this issue. Please use the workarounds in the meantime

**Comment by mpstein** [comment]
I'm pretty sure you're using this issues board to keep track of your own roadmap, but I figure I'll keep working on random things like this to help out where I can.

The issue is that the Foundry app is running as root, which is generally considered a bad practice. This is being set in the service definition located in `/etc/systemd/system/foundry.service`

I'm going to play around with this a little bit as it might just be simpler to create a new `foundry` user and group and set the new group as a supplemental one for the ec2-user. I'll play around with it a bit and see if I can clean this up for you.

**Comment by cat-box** [comment]
@mpstein that's exactly what's going on lol

we had planned on creating a new user entirely to fix the issue but neither of us have had the time to actually sit down and fully address it yet. 

I was planning on getting to it next week but if you'd like to fix it before then, have at it :)

**Comment by PupperooX** [comment]
Hey, so I've run into this same issue in that I can't upload anything because my ec2-user doesn't have access. Is there a good fix for it that I can implement asap? (I'm very new to it so I'm not sure how to dig around)

**Comment by mpstein** [comment]
Heya! I stepped away from this project as I changed the way I was hosting Foundry, but let me see if I can give you some functional instructions to help you out.

There are a few ways to do things, but all require a bit of comfort with the command line interface. In my opinion, the easiest way of solving your issue is uploading all of your content to the ec2-user home directory, becoming root, changing ownership, and moving files over. 

@PupperooX , are you uploading your files through Foundry (or via FTP, SCP, or rsync)? I don't have time to work on an automated fix this morning, but I might be able to help you with a workaround.
